sci-geosciences/grass/grass-8.4.1.ebuild

```diff
@@ -0,0 +1,294 @@
+	readline? ( sys-libs/readline:= )
+	sqlite? ( dev-db/sqlite:3 )
+	svm? ( sci-libs/libsvm:= )
+	tiff? ( media-libs/tiff:= )
+	truetype? ( media-libs/freetype:2 )
+	X? (
+		$(python_gen_cond_dep '
+			>=dev-python/matplotlib-1.2[wxwidgets,${PYTHON_USEDEP}]
+			dev-python/pillow[${PYTHON_USEDEP}]
+			>=dev-python/wxpython-4.1:4.0[${PYTHON_USEDEP}]
+		')
+		x11-libs/cairo[X]
+		x11-libs/libICE
+		x11-libs/libSM
+		x11-libs/libX11
+		x11-libs/libXext
+		x11-libs/libXt
+	)
+	zstd? ( app-arch/zstd:= )"
+DEPEND="${RDEPEND}
+	X? ( x11-base/xorg-proto )"
+BDEPEND="
+	app-alternatives/yacc
+	app-alternatives/lex
+	sys-devel/gettext
+	virtual/pkgconfig
+	X? ( dev-lang/swig )"
+
+PATCHES=(
+	# bug 746590
+	"${FILESDIR}/${PN}-flock.patch"
+)
+
+pkg_pretend() {
+	[[ ${MERGE_TYPE} != binary ]] && use openmp && tc-check-openmp
+}
+
+pkg_setup() {
+	[[ ${MERGE_TYPE} != binary ]] && use openmp && tc-check-openmp
+
+	if use lapack; then
+		local mylapack=$(eselect lapack show)
+		if [[ -z "${mylapack/.*reference.*/}" ]] && \
+			[[ -z "${mylapack/.*atlas.*/}" ]]; then
+			ewarn "You need to set lapack to atlas or reference. Do:"
+			ewarn "   eselect lapack set <impl>"
+			ewarn "where <impl> is atlas, threaded-atlas or reference"
+			die "setup failed"
+		fi
+	fi
+
+	if use blas; then
+		local myblas=$(eselect blas show)
+		if [[ -z "${myblas/.*reference.*/}" ]] && \
+			[[ -z "${myblas/.*atlas.*/}" ]]; then
+			ewarn "You need to set blas to atlas or reference. Do:"
+			ewarn "   eselect blas set <impl>"
+			ewarn "where <impl> is atlas, threaded-atlas or reference"
+			die "setup failed"
+		fi
+	fi
+
+	python-single-r1_pkg_setup
+}
+
+src_prepare() {
+	# Fix unversioned python calls
+	sed -e "s:=python3:=${EPYTHON}:" -i "${S}/lib/init/grass.sh" || die
+	sed -e "s:= python3:= ${EPYTHON}:" -i "${S}/include/Make/Platform.make.in" || die
+
+	default
+
+	# When patching the build system, avoid running autoheader here. The file
+	# config.in.h is maintained manually upstream. Changes to it may lead to
+	# undefined behavior. See bug #866554.
+	# AT_NOEAUTOHEADER=1 eautoreconf
+
+	ebegin "Fixing python shebangs"
+	python_fix_shebang -q "${S}"
+	eend $?
+
+	# For testsuite, see https://bugs.gentoo.org/show_bug.cgi?id=500580#c3
+	local ati_cards mesa_cards nvidia_cards render_cards
+	local prev_shopt=$(shopt -p nullglob)
+	shopt -s nullglob
+	ati_cards=$(echo -n /dev/ati/card*)
+	for card in "${ati_cards[@]}"; do
+		addpredict "${card}"
+	done
+	mesa_cards=$(echo -n /dev/dri/card*)
+	for card in "${mesa_cards[@]}"; do
+		addpredict "${card}"
+	done
+	nvidia_cards=$(echo -n /dev/nvidia*)
+	for card in "${nvidia_cards[@]}"; do
+		addpredict "${card}"
+	done
+	render_cards=$(echo -n /dev/dri/renderD128*)
+	for card in "${render_cards[@]}"; do
+		addpredict "${card}"
+	done
+	addpredict /dev/nvidiactl
+	${prev_shopt}
+}
+
+src_configure() {
+	# -Werror=strict-aliasing
+	# https://bugs.gentoo.org/862579
+	# https://github.com/OSGeo/grass/issues/3506
```

Comment:
(Would you mind checking the suggestion on the upstream bug btw, so we can drop these?)

Reply:
Unfortunately I failed to reproduce the initial bug (even with old versions of the package). Thus I am unable to prove if the flags are still necessary. I feel too risky in dropping this statement under these circumstances.